crates/rooch-types/src/genesis_config.rs

```diff
@@ -150,24 +150,32 @@ pub static G_TEST_CONFIG: Lazy<GenesisConfig> = Lazy::new(|| {
     }
 });
 
+//curl -sSL https://mempool.space/api/block/$(curl -sSL https://mempool.space/api/block-height/852203)/header
+
+static MAIN_GENESIS_HEIGHT_HEADER: Lazy<(u64, Header)> = Lazy::new(|| {
+    (852203, bitcoin::consensus::deserialize(
```

Comment:
Shouldn't this be 852202?

Reply:
The Bitcoin block will execute from 852203, include 852203